interfaces/builtin/libvirt.go

```diff
@@ -33,6 +33,7 @@ const libvirtConnectedPlugAppArmor = `
 /run/libvirt/libvirt-sock-ro rw,
 /run/libvirt/libvirt-sock rw,
 /etc/libvirt/* r,
+/var/lib/snapd/hostfs/var/lib/libvirt/dnsmasq{,**} r,
```

Comment:
did you mean this?
```suggestion
/var/lib/snapd/hostfs/var/lib/libvirt/dnsmasq/{,**} r,
```

Reply:
Absolutely, fixed in https://github.com/snapcore/snapd/pull/13645/commits/6638f7e1ffa24d405f797bba387cdb569282098a